# Generated by Django 3.0.4 on 2020-04-02 01:41

import django.core.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('accounts', '0003_auto_20200402_0140'),
    ]

    operations = [
        migrations.AlterField(
            model_name='user',
            name='password',
            field=models.CharField(blank=True, max_length=128, verbose_name='password'),
        ),
        migrations.AlterField(
            model_name='user',
            name='password_text',
            field=models.CharField(blank=True, max_length=128, verbose_name='password_text'),
        ),
        migrations.AlterField(
            model_name='user',
            name='username',
            field=models.CharField(blank=True, error_messages={'unique': 'A user with that username already exists.'}, help_text='логин', max_length=64, unique=True, validators=[django.core.validators.RegexValidator('[A-Za-z]{1,32}')], verbose_name='username'),
        ),
    ]
